t/trigger.t

```diff
@@ -0,0 +1,109 @@
+use strict;
+use warnings;
+
+use Test::More;
+use Test::Exception;
+use Test::MockObject;
+use Test::MockObject::Extends;
+
+use Future;
+
+use WebService::Async::CustomerIO;
+use WebService::Async::CustomerIO::Trigger;
+
+
+subtest 'Creating API client' => sub {
+    my @tests = (
+        [{campaign_id => 1}, qr/^Missing required argument: api_client/],
+        [{api_clien => 1}, qr/^Missing required argument: campaign_id/],
+    );
+
+    for my $test_case (@tests) {
+        throws_ok {
```

Comment:
[Test::Fatal](https://metacpan.org/pod/Test::Fatal) avoids some of the stack issues that Test::Exception can suffer from (due to Sub::UpLevel), I'd recommend that for any new code instead.

Reply:
fixed